deploy-disconnected-registry/olm-sync.sh

```diff
@@ -97,6 +97,15 @@ function mirror() {
     sed -i '/^mountopt =.*/d' /etc/containers/storage.conf
     #######
 
+    ####### WORKAROUND: Fixes "Invalid GPG signature for image certified-operator-index"
+    ####### https://access.redhat.com/solutions/6542281
```

Comment:
Can you describe in this comment what the 2 `jq` commands do? Just for the sake of future us forgetting what this was supposed to do.

Reply:
refactored to a single command and added a comment.